src/invoices/utils.js

```diff
@@ -1,4 +1,13 @@
-import { find } from 'lodash';
+import { find, invert } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AmountWithCurrencyField,
+  ORDER_STATUS_LABEL,
+  PAYMENT_STATUS,
+  RECEIPT_STATUS,
+} from '@folio/stripes-acq-components';
+import { NoValue } from '@folio/stripes/components';
```

Comment:
should be imported above `stripes-acq-components`

Reply:
updated